epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -20,21 +21,30 @@ public EpoxyViewHolder(ViewGroup parent, @LayoutRes int layoutId) {
   public void bind(@SuppressWarnings("rawtypes") EpoxyModel model, List<Object> payloads) {
     this.payloads = payloads;
 
+    if (epoxyHolder == null && model instanceof EpoxyModelWithHolder) {
+      epoxyHolder = ((EpoxyModelWithHolder) model).createNewHolder();
+      epoxyHolder.bindView(itemView);
+    }
+
     if (payloads.isEmpty()) {
       // noinspection unchecked
-      model.bind(itemView);
+      model.bind(objectToBind());
```

Comment:
No, unfortunately. As far as I know at least. It would mean typing the View holder with the view type, which isn't possible because the view types are all dynamic.

It doesn't really matter though since this is safe and not going to be the wrong type, and it's hidden in the library so the casting isn't exposed to the user.